ibm_catalog.json

```diff
@@ -250,217 +233,340 @@
               }
             },
             {
-              "key": "cos_instance_access_tags",
-              "custom_config": {
-                "grouping": "deployment",
-                "original_grouping": "deployment",
-                "config_constraints": {
-                  "type": "string"
-                }
-              }
+              "key": "global_resource_group_name"
             },
             {
-              "key": "cos_plan",
-              "options": [
-                {
-                  "displayname": "Standard",
-                  "value": "standard"
-                },
-                {
-                  "displayname": "COS Onerate Plan",
-                  "value": "cos-one-rate-plan"
-                }
-              ]
+              "key": "use_existing_global_resource_group"
+            },
+            {
+              "key": "audit_resource_group_name"
+            },
+            {
+              "key": "use_existing_audit_resource_group"
+            },
+            {
+              "key": "devops_resource_group_name"
+            },
+            {
+              "key": "use_existing_devops_resource_group"
+            },
+            {
+              "key": "edge_resource_group_name"
+            },
+            {
+              "key": "use_existing_edge_resource_group"
+            },
+            {
+              "key": "management_resource_group_name"
+            },
+            {
+              "key": "use_existing_management_resource_group"
+            },
+            {
+              "key": "observability_resource_group_name"
+            },
+            {
+              "key": "use_existing_observability_resource_group"
+            },
+            {
+              "key": "security_resource_group_name"
+            },
+            {
+              "key": "use_existing_security_resource_group"
+            },
+            {
+              "key": "workload_resource_group_name"
+            },
+            {
+              "key": "use_existing_workload_resource_group"
+            },
+            {
+              "key":"cbr_enforcement_mode"
             },
             {
-              "key": "cos_bucket_name"
+              "key":"provision_cbr"
             },
             {
-              "key": "cos_bucket_access_tags",
+              "key": "cbr_kms_service_targeted_by_prewired_rules",
               "custom_config": {
                 "grouping": "deployment",
                 "original_grouping": "deployment",
                 "config_constraints": {
                   "type": "string"
                 }
               }
-            },
+            }
+          ]
+        },
+        {
+          "label": "Resource group only",
+          "name": "resource-group-only",
+          "install_type": "fullstack",
+          "working_directory": "solutions/account-infrastructure-base",
+          "compliance": {
```

Comment:
I don't think there are any relevant SCC rules for resource group creation, so I would leave this out of this variation